adapter/src/main/resources/Dockerfile.ubuntu

```diff
@@ -43,13 +43,15 @@ RUN \
     groupadd --system -g ${MG_USER_GROUP_ID} ${MG_USER_GROUP} \
     && useradd --system --create-home --home-dir ${MG_USER_HOME} --no-log-init -g ${MG_USER_GROUP_ID} -u ${MG_USER_ID} ${MG_USER} \
     && mkdir ${MG_USER_HOME}/logs && chown -R ${MG_USER}:${MG_USER_GROUP} ${MG_USER_HOME}/logs  \
+    && mkdir -p ${MG_USER_HOME}/artifacts/apis \
```

Comment:
In line 45 we have specifically provided the relevant permission to the logs directory. Can you check the exact permission after this change? I think we need to do the relevant permissions.

Reply:
yes, since we are running the the adapter process under the user id `wso2` and group id `wso2`, we need to set relevant permissions. Added `chown -R ${MG_USER}:${MG_USER_GROUP} ${MG_USER_HOME}/artifacts`